lib/3scale/backend/stats/service_context.rb

```diff
@@ -0,0 +1,14 @@
+# Create a keys partition generator
+module ThreeScale
+  module Backend
+    module Stats
+      class ServiceContext
```

Comment:
minor issue, but I would not call this service context. This is the removal job description.